src/config/.npmpackagejsonlintrc.json

```diff
@@ -5,14 +5,15 @@
       "require-bugs": "error",
       "require-bundledDependencies": "off",
       "require-config": "off",
-      "require-contributors": "off",
+      "require-contributors": "error",
```

Comment:
This will error when the contributors field is absent, right?  ipfs has [contributors in the root package.json](https://github.com/ipfs/js-ipfs/blob/master/package.json#L68-L255) but not always in the modules under `/packages` as otherwise we'd have to compare the paths of edits during our `update-contributors` step.